content/attacks/posts/2023-04-02-Allbridge.md

```diff
@@ -0,0 +1,39 @@
+---
+date: 2023-04-02
+target-entities: 
+- Allbridge
+entity-types:
+- DeFi
+- Bridge
+attack-types:
+- Smart Contract Exploit
+- Flash Loan Attack
+title: "A Close Look into the $573K Allbridge Exploit and Its Resolution"
```

Comment:
Title should be describing, for example: `Allbridge suffered a flash loan attack for over $500k`